rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -909,7 +912,7 @@ bool game_list_frame::DeleteSPUCache(const std::string& base_dir, bool is_intera
 	return true;
 }
 
-QPixmap game_list_frame::PaintedPixmap(const QImage& img, bool paint_config_icon)
+QPixmap game_list_frame::PaintedPixmap(const QImage& img, bool paint_config_icon, QColor compatibility_color)
```

Comment:
const QColor&

Reply:
I change the alpha value of the color later on, so setting the color to const would require me to copy the color into a new variable either way. But maybe that'd be better when the color is invalid, I'll think about it.